scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryPartitionUtil.scala

```diff
@@ -58,7 +58,7 @@ private[bigquery] object BigQueryPartitionUtil {
   private def getPartitions(bq: BigQuery, tableRef: TableReference): Set[String] = {
     val prefix = tableRef.getTableId.split('$')(0)
     bq.tables
-      .tableReferences(tableRef.getProjectId, tableRef.getDatasetId)
+      .tableReferences(tableRef.getProjectId(), tableRef.getDatasetId())
```

Comment:
```suggestion
      .tableReferences(tableRef.getProjectId, tableRef.getDatasetId)
```

Reply:
nit: this is unnecessary? IntelliJ won't add those `()`s

Reply:
yeah intellij likes that :man_shrugging: ... regardless the idea is actually not to change this line ... missed that.